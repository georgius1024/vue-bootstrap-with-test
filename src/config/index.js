let production = {
  APP_ID: 'bbt-web',
  APP_NAME: 'Большие Банковские Тесты',
  APP_VERSION: '5.0.0.0 pre',
  APP_COPYRIGHT: '© 1999, 2018 Агентство ВЭП',
  API_URL: '/api/'
}

let development = Object.assign({}, production, {
  API_URL: 'http://localhost:3800/'
})

let test = Object.assign({}, development)

let config = {
  development,
  test,
  production
}

let env = process.env.NODE_ENV || 'development'
export default config[env]
