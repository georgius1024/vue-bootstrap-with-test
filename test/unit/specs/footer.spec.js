import config from  '@/config'
import {
  shallowMount
} from '@vue/test-utils'

import Footer from '@/components/app/footer'

describe('footer.vue', () => {
  it('should render correct contents', () => {
    const wrapper = shallowMount(Footer, {
      propsData: {
        copyright: config.APP_COPYRIGHT,
        application: config.APP_NAME,
        version: config.APP_VERSION
      }
    })
    expect(wrapper.html()).toContain(config.APP_COPYRIGHT)
    expect(wrapper.html()).toContain(config.APP_NAME)
    expect(wrapper.html()).toContain(config.APP_VERSION)
  })

})
