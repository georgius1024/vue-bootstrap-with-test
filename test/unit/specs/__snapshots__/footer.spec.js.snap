// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer.vue should render correct contents 1`] = `
<v-footer-stub
  absolute=""
  app=""
  dark=""
  height="32"
>
  <div
    class="mx-4"
  >
    © 1999, 2018 Агентство ВЭП
  </div>
   
  <v-spacer-stub />
   
  <div
    class="mr-4"
  >
    Большие Банковские Тесты v5.0.0.0 pre
  </div>
</v-footer-stub>
`;
